src/lnd-setup/generate-lightning-client.js

```diff
@@ -60,7 +65,9 @@ function generateLightningClient ({ host, protoPath, tlsCertPath, macaroonPath,
     rpcCredentials = grpc.credentials.combineChannelCredentials(sslCredentials, macaroonCredentials)
   }
 
-  return new lnrpc.Lightning(host, rpcCredentials, {})
+  const client = new lnrpc.Lightning(host, rpcCredentials, {})
```

Comment:
is this kept as the base client to prevent too many cascading changes in other parts of this and other repos?

Reply:
Yeah, it was causing trouble the other way and I figured that could be a separate refactoring if it's considered important enough.